internal/cmd/images.go

```diff
@@ -0,0 +1,86 @@
+package cmd
+
+import (
+	"encoding/json"
+	"os"
+
+	"cdr.dev/coder-cli/coder-sdk"
+	"cdr.dev/coder-cli/pkg/clog"
+	"cdr.dev/coder-cli/pkg/tablewriter"
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+)
+
+func imgsCmd() *cobra.Command {
+	var user string
+
+	cmd := &cobra.Command{
+		Use:   "images",
+		Short: "Manage Coder images",
+		Long:  "Perform operations on the Coder images owned by the active user.",
+	}
+
+	cmd.PersistentFlags().StringVar(&user, "user", coder.Me, "Specify the user whose resources to target")
+	cmd.AddCommand(lsImgsCommand(&user))
+	return cmd
+}
+
+func lsImgsCommand(user *string) *cobra.Command {
+	var (
+		orgName   string
+		outputFmt string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "ls",
+		Short: "list all images owned by the active user",
+		Long:  "List all Coder images owned by the active user.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
+			client, err := newClient(ctx)
+			if err != nil {
+				return err
+			}
+
+			imgs, err := getImgs(ctx, client,
+				getImgsConf{
+					email:   *user,
+					orgName: orgName,
+				},
+			)
+
+			if err != nil {
+				return err
+			}
+
+			if len(imgs) < 1 {
+				clog.LogInfo("no images found")
+				imgs = []coder.Image{} // ensures that json output still marshals
+			}
+
+			if outputFmt == jsonOutput {
```

Comment:
We should use a `switch` here to ensure we error on invalid flag values and that we can easily add new output types.